starknet_py/net/account/account_client.py

```diff
@@ -161,12 +166,19 @@ async def add_transaction(
     async def estimate_fee(
         self,
         tx: InvokeFunction,
+        block_hash: Optional[CastableToHash] = None,
+        block_number: BlockIdentifier = "pending",
     ) -> int:
+        # pylint: disable=duplicate-code
         """
         :param tx: Transaction which fee we want to calculate
+        :param block_hash: Estimate fee at specific block hash
+        :param block_number: Estimate fee at given block number (or "pending" for pending block)
         :return: Estimated fee
         """
-        return await super().estimate_fee(await self._sign_transaction(tx))
+        return await super().estimate_fee(
+            await self._sign_transaction(tx), block_hash, block_number
```

Comment:
I think it'd be better to use keyword arguments like in Client `block_hash=block_hash...`. Much less error prone this way.